pkg/config.go

```diff
@@ -98,6 +98,10 @@ func saveCookies(c *http.Client, u *url.URL, config *Config) error {
 }
 
 func parseResolvConf(config *Config) {
+	if config.resolvConf == nil {
```

Comment:
Good move :) If it is `nil`, there was no `resolv.conf`, if it is just empty, the _existing_ `resolv.conf` was empty.